assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -128,34 +131,43 @@ module netcdf_utilities_mod
 
 interface nc_put_variable
    module procedure nc_put_char_1d
-   module procedure nc_put_single_int_1d
-   module procedure nc_put_int_1d
-   module procedure nc_put_single_real_1d
-   module procedure nc_put_real_1d
```

Comment:
the 'nc_put_single_xxxx' procedures are renamed `nc_put_xxx_0d` in keeping with the 1d,2d,3d, notation.

Reply:
see my other comment for why these are actually different routines, and if the interface overloading still works, that we might need both.

Reply:
Understood ... but I'm not sure we need a routine to stick a scalar (as opposed to a length-1 array) into a particular slot in a netCDF array variable. We do not do that  in any of the code tested by test_dart.csh

Reply:
agreed - now that you pointed out that it was only used in a single place (which has been changed to put the entire array) i'm happy to remove that variant.  (and sorry - i should have opened a review on this instead of nickel and dime-ing my comments.